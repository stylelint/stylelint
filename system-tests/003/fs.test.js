'use strict';

const stylelint = require('../../lib');
const { caseConfigFile, caseFilesForFix, prepForSnapshot } = require('../systemTestUtils');

const CASE_NUMBER = '003';

it('no-fs - zen garden CSS with standard config', async () => {
	expect(
		prepForSnapshot(
			await stylelint.lint({
				files: await caseFilesForFix(CASE_NUMBER),
				configFile: caseConfigFile(CASE_NUMBER),
				fix: true,
			}),
		),
	).toMatchSnapshot();
}, 10000);
