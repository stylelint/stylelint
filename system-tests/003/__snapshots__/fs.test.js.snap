// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-fs - zen garden CSS with standard config 1`] = `
Object {
  "errored": true,
  "output": Array [
    Object {
      "deprecations": Array [],
      "errored": true,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [
        Object {
          "column": 25,
          "line": 102,
          "rule": "font-family-no-missing-generic-family-keyword",
          "severity": "error",
          "text": "Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)",
        },
      ],
    },
  ],
  "results": Array [
    Object {
      "deprecations": Array [],
      "errored": true,
      "ignored": undefined,
      "invalidOptionWarnings": Array [],
      "parseErrors": Array [],
      "warnings": Array [
        Object {
          "column": 25,
          "line": 102,
          "rule": "font-family-no-missing-generic-family-keyword",
          "severity": "error",
          "text": "Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)",
        },
      ],
    },
  ],
}
`;
